t/lib/MemcachedTest.pm

```diff
@@ -236,7 +237,8 @@ sub new_memcached {
             $conn = eval qq{IO::Socket::SSL->new(PeerAddr => "$host:$port",
                                         SSL_verify_mode => SSL_VERIFY_NONE,
                                         SSL_cert_file => '$client_crt',
-                                        SSL_key_file => '$client_key');
+                                        SSL_key_file => '$client_key',
+                                        SSL_ca_file => '$ca_crt');
```

Comment:
I would not add CA_cert to defaults (when SSL is enabled) as it's an optional parameter. You may use `https://github.com/memcached/memcached/blob/master/t/ssl_verify_modes.t#L17` in `ssl_cert_refresh.t`